### ISSUE TITLE
github: enable polls apps on yaml organization

### ISSUE THREAD

**Issue by ssbarnea** [yaml/yaml#55]
I would like to request enabling https://github.com/apps/polls on the yaml organization in order to allow us to add polls to tickets.

Polls look like https://github.com/ansible-community/molecule/issues/2590 and allow github users to vote by clicking on the bar. For security reasons a poll question cannot be altered after creation as this would reset all previous votes. Just be careful how we phrase the polls in order to avoid this, avoiding bias and including a "don't care" kind of answer, that should cover those that do not mind whatever option is picked.

The first poll/survey, I would like to ask is related to preference about sequence indentation in yaml files.

**Comment by nutcrack** [comment]
good idea. As yaml community grows, more features are foreseen to be a very nice complementary for yaml syntax, and i'm sure more than two people would thumb this up.
